pytorch_blade/tests/disc/pdl/test_torchscipt_to_mhlo/test_quantization.py

```diff
@@ -63,7 +63,8 @@ def forward(self, x):
                     self.activation_quant_min, self.activation_quant_max
                 )
                 return x
-
+        if self.device == torch.device('cuda'):
```

Comment:
It is better to build different base classes for different device (e.g. gpu and cpu) to enable testing on the respective hardware. Also, skip test using `return` may not be a good idea, use the api of `unitest` instead

Reply:
Have rebased. Also add the following rewrite pattern for gpu kernel implementation. The equality between original graph and rewritten graph is not strict. 
    #  weight -> fake-quant  \
    #  input  -> fake-quant -> linear -> fake-quant -> output
    #                   bias /
    #
    # rewrited graph:
    #
    #  weight -> quant  \
    #  input  -> quant -> qgemm -> dequant -> output
    #  bias   -> quant  /

Reply:
remove this codes?

Reply:
done